Source/WTF/wtf/SequesteredAllocator.h

```diff
@@ -46,7 +46,7 @@
 
 namespace WTF {
 
-constexpr size_t minArenaGranuleSize { 16 * KB };
+constexpr size_t minArenaGranuleSize { 16 * 16 * KB };
```

Comment:
Why is this needed?

Reply:
Improves performance -- since we leave the top granule in place, this is essentially the maximum a compiler job can allocate without needing to mmap